javascript/javascript1/week4/voice-assistant.js

```diff
@@ -0,0 +1,189 @@
+"use strict";
+
+//Creating global scope variables
+let userName = "";
```

Comment:
Is there any reason to make this a global scope?

Reply:
Thanks a lot for the feedback. I fixed variable declaration (let -> const) in the cases you mentioned - everything works perfectly. I also deleted breaks in the switch statement as you said - I don't need them if I have returns. 

As for userName as a global scope, I tried moving it inside the function, but some of the features (1 - when a user says his name again and the assistant answers with "Hi again"; 2 - when a user asks for the name) stopped working. Maybe there is a way out of it.